web/maestro-ui/README.md

```diff
@@ -0,0 +1,23 @@
+# Maestro Node UI (Dev/Prod)
+
+## Dev (Vite)
+
+- Prereqs: Python venv active, backend config (e.g., OPENAI_API_KEY)
+- Start: `maestro deploy agents.yaml workflow.yaml --node-ui --ui-mode dev`
```

Comment:
This doesn't work: `sh: vite: command not found`

I assume there's a build step that needs to happen first but is not included in this doc, but that would also mean new build steps to run this in the top level, as the command should work out of the box after pip install (uv sync in dev)